docs/source/deploying-volttron/proxy_web_server.rst

```diff
@@ -0,0 +1,305 @@
+
+*********************************************
+Reverse Proxy Example for VOLTTRON Deployment
+*********************************************
+
+The purpose of this section is to review the necessary dependencies and configurations for setting up and deploying
+a reverse proxy for VOLTTRON using apache2 as the proxy on Ubuntu.
+
+Configuring VOLTTRON
+====================
+
+For this setup, VOLTTRON should be web enabled, using the http protocol and default web port of 8080. The
+bind-web-address and volttron-central-address should be set to localhost (http://localhost:8080).
+The proxy will handle requiring https. Refer to :ref:`this section <VOLTTRON-Central-Deployment>` for guidance
+on setting up an instance of VOLTTRON Central.
+
+Install Dependencies
+====================
+
+To set up a reverse proxy using apache for VOLTTRON, install apache2
+and enable the ssl, rewrite, headers, proxy, and proxy_http modules.
+This will require sudo access and a restart of the apache2 service.
+
+.. code-block::
+
+    sudo apt-get install apache2
+    sudo a2enmod ssl
+    sudo a2enmod rewrite
+    sudo a2enmod headers
+    sudo a2enmod proxy
+    sudo a2enmod proxy_http
+    sudo service apache2 restart
+
+
+Configuring Apache2
+===================
+
+In ubuntu, the apache configuation is divided into a series of directories. For setting up a basic reverse proxy
+for VOLTTRON, there are three that we need to modify, apache2.conf, ssl.conf, and 000-default.conf.
+
+
+apache2.conf File
+-----------------
+
+The apache2.conf sits in the root apache2 directory and serves as a top level configuration file
+for the apache2 web server. This is the main Apache server configuration file. It contains the configuration directives
+that give the server its instructions. Debian's apache2 configuration is split into several files forming a
+configuration hierarchy. The apache2.conf file puts the pieces together by including all
+remaining configuration files when starting up the web server. The default location for this file is at
+/etc/apache2/apache2.conf.
+
+An example apache2.conf config file can be found within the volttron repository at
+volttron/scripts/admin/apache-proxy/apache2.conf. Within the file, you will need to set the
+User and Group options, with ServerAdmin being optional.
+
+.. code-block:: console
+
+    # apache2.conf
+
+    ServerRoot "/etc/apache2"
+
+    # Set User, Group, and ServerAdmin based on your system and organization
+
+    # User will be a non-root unix user that the server will use to respond to requests.
+    # It is recommended to create a new user group specifically for the apache server.
+    # The user and group should not have the ability to access any files that are not
+    # intended to be available through the VOLTTRON web service, or have the ability to
+    # execute code beyond the anticipated scope. This user does not need permissions for
+    # all of VOLTTRON, only the web hosted materials.
+    User <user name>
```

Comment:
the apache server must be able to write to the logs directory you reference in the troubleshooting section.  Specifying the user and group here will allow the server to be run as the user specified, but will not change permissions on files/directories.

Reply:
Leave out the part of volttron in this note. As this is about the apache not volttron.

Reply:
Focused context on apache2 proxy. Logs are written by root for the service, the user, group settings are not used for writing logs.